restconnector/RequestHandler.cpp

```diff
@@ -113,7 +127,16 @@ void RequestHandler::onImageJPEGEvent_( const ::zeq::Event& event )
 
 void RequestHandler::onHeartbeatEvent_()
 {
-    LBINFO << "Heartbeat event received." << std::endl;
+    if ( renderingRessourceStrarted_ )
```

Comment:
What do you mean by "deregistering this method" ? Remove this method? We need this method to be able to detect a Rendering Ressource crash ( crash detection behaviour will be implemented soon )